tests/introspection/iOS/iOSApiProtocolTest.cs

```diff
@@ -61,6 +61,15 @@ protected override bool Skip (Type type)
 					return true;
 
 				break;
+			case "MTLFence":
+			case "MTLHeap":
+				if (Runtime.Arch != Arch.DEVICE)
```

Comment:
In this case, we have the following, if we are not in the device, return true. Else, looks at like 70 and checks that the Xcode version is 8,0, else return true. Meaning that it should be skipped when we are not in the device or when the Xcode version is not 8.

Exact same logic as the following:

```csharp
// Apple does not ship a PushKit for every arch on some devices :(
case "PKPushCredentials":
case "PKPushPayload":
case "PKPushRegistry":
    if (Runtime.Arch != Arch.DEVICE)
        return true;
    // Requires iOS 8.2 or later in 32-bit mode
    if (!TestRuntime.CheckXcodeVersion (6, 2) && IntPtr.Size == 4)
        return true;

	break;
```